resources/views/layouts/app.blade.php

```diff
@@ -3,8 +3,10 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="{{ asset('css/app.css') }}">
-    </link>
+    <!-- <link rel="stylesheet" href="{{ asset('css/app.css') }}">
+    </link> -->
+    <link rel="stylesheet" href="http://ii.test/css/app.css">
+    <link rel="stylesheet" href="/css/app.css">
```

Comment:
It was better before. Don't change your code permanently just so you can use ngrok once in a while. Also: Link tags don't need a closing tag.

Reply:
already changed it. thanks.